packages/core/src/alert/alert-actions.element.ts

```diff
@@ -71,23 +71,28 @@ function updateLayoutsAndActionButtonStyles(alertActionsComponent: CdsAlertActio
  *
  * ```html
  * <cds-alert>
- *    Lorem ipsum dolor sit amet
- *    <cds-alert-actions>
- *       <cds-button>Fix</cds-button>
- *    </cds-alert-actions>
+ *   Lorem ipsum dolor sit amet
+ *   <cds-alert-actions>
+ *     <cds-button>Fix</cds-button>
+ *   </cds-alert-actions>
  * </cds-alert>
  * ```
  *
  * @beta
  * @element cds-alert-actions
  * @slot default
- * @cssprop --action-text-color: changes the color of the text and border of the action button
- * @cssprop --action-hover-text-color: changes the color of the text and border of the action button on hover
+ * @cssprop --action-text-color
```

Comment:
Added missing API docs. The description on the prop was breaking the generation of the api json.